bolt/test/AArch64/jump-table-info.s

```diff
@@ -0,0 +1,236 @@
+## Check parsing of a .llvm_jump_table_info section
```

Comment:
(Can use the llvm/utils/update_test_body.py format to make re-generation easier :) )